x-pack/plugins/endpoint/public/lib/action_creator.test.ts

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
❔  Q. regarding `/lib/` directory - should it be placed directly under `public`? or move to `applications/endpoint`

Reply:
i think it should be under `applications/endpoint`. or we could move `applications/endpoint` to `application` and move it there. i'd rather keep the Endpoint app and Resovler just slightly separated. For example, this pr adds a global types module, but the types will be different vs Resolvers types.

Reply:
Thanks. I will move the `lib/` and `types.ts` to `application/endpoint`.